smdebug/tensorflow/base_hook.py

```diff
@@ -421,6 +421,11 @@ def set_gradients(self, gradients=None, gradients_and_variables=None):
         # Skipping set_gradients and set_optimizer_variables for Tf 2.x until there is
         # support to pass names and values from TF side.
         if is_tf_version_2x() and tf.executing_eagerly():
+            self.logger.info(
+                "TF 2.x doesn't provide gradient and optimizer variable values."
```

Comment:
how frequently will this be printed. One time, or every n steps?

Reply:
every step. do we need a log here? the comment has been added to take care of this case in the near future when support is added to get gradient values apart from name

Reply:
every step is bad. Can we just log in the begining that this is TF 2.x and executing eagerly, blah blah is not supported.

Reply:
addressed in the latest commit